src/entries/popup/pages/swap/SwapTokenInput/TokenDropdown/TokenToBuyDropdown.tsx

```diff
@@ -34,46 +33,44 @@ export const TokenToBuyDropdown = ({
   setOutputChainId,
   onDropdownChange,
 }: TokenToBuyDropdownProps) => {
-  const isL2 = useMemo(() => isL2Chain(outputChainId), [outputChainId]);
+  const isL2 = useMemo(
+    () => outputChainId && isL2Chain(outputChainId),
+    [outputChainId],
+  );
+
   const assetsCount = useMemo(
     () => assets?.reduce((count, section) => count + section.data.length, 0),
     [assets],
   );
 
+  const t = useTranslationContext();
```

Comment:
i feel like this is gonna be painful to maintain in future changes

for example, `explainers.flashbots` doesn't have any difference between swaps and bridges (most of the translations don't), adding that we'll have to maintain all of the languages

why not just change the swap / bridging related text and leave the rest as it was?

i'm imagining having some text like

```
{
  select_tokens `Select tokens to %{action}`
}
```

and that `action` could come from the `useTranslationContext` hook or something like that

wdyt?

Reply:
yes I was concerned with that too, I'll try this action idea